docs/index.rst

```diff
@@ -257,10 +256,7 @@ following papers:
    Communications Security 2006 (CCS 2006), Alexandria, Virginia,
    October, 2006
 
-Past contributors: Khoo Yit Phang, Ed Schwartz, Mike Katelman (PhD
-Student, University of Illinois, Urbana-Champaign, IL, USA), Philip Guo
-(Student, Stanford University, Stanford, CA, USA), David L. Dill
-(Professor, Stanford University, Stanford, CA, USA), Tim King (Student,
-Stanford University and NYU). Please note that everyone working on the
-project is doing so out of hobby or as a way to help them in their
-work/study projects.
+Contributors are listed on the `contributions page <https://github.com/stp/stp/graphs/contributors>`__.
+
+In addition we thank Ed Schwartz, Mike Katelman, Philip Guo, David L. Dill, and Tim King. Please note that everyone working on the
```

Comment:
This is such a nit: can we wrap this at 80 chars (or whatever the rest of the file is wrapped at)?

Reply:
I've fixed up the word wrap.